Shared/Samples/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotationView.Model.swift

```diff
@@ -0,0 +1,170 @@
+            return map
+        }()
+        
+        /// A URL to the temporary file containing the geodatabase.
+        private let temporaryGeodatabaseURL = FileManager
+            .createTemporaryDirectory()
+            .appending(component: "LoudounAnno.geodatabase")
+        
+        /// The building number and street name of the selected feature.
+        var selectedFeatureAddress: (buildingNumber: Int32, streetName: String)? {
+            if let buildingNumber = selectedFeature?.attributes[.addressFieldKey] as? Int32,
+               let streetName = selectedFeature?.attributes[.streetNameFieldKey] as? String {
+                return (buildingNumber, streetName)
+            } else {
+                return nil
+            }
+        }
+        
+        deinit {
+            // Removes the temporary geodatabase file and its directory.
+            let temporaryDirectoryURL = temporaryGeodatabaseURL.deletingLastPathComponent()
+            try? FileManager.default.removeItem(at: temporaryDirectoryURL)
+        }
+        
+        /// Adds feature and annotation layers created from geodatabase feature tables to the map.
+        func setUpMap() async throws {
+            // Creates a geodatabase from a file copied from the bundle.
+            try FileManager.default.copyItem(at: .loudounAnnoGeodatabase, to: temporaryGeodatabaseURL)
+            let geodatabase = Geodatabase(fileURL: temporaryGeodatabaseURL)
+            try await geodatabase.load()
+            
+            // Creates feature and annotation layers from tables in the geodatabase.
+            let featureTableNames = ["ParcelLines_1", "Loudoun_Address_Points_1"]
+            let featureLayers = featureTableNames
+                .compactMap { geodatabase.featureTable(named: $0) }
+                .map(FeatureLayer.init)
+            
+            let annotationTableNames = ["ParcelLinesAnno_1", "Loudoun_Address_PointsAnno_1"]
+            let annotationLayers = annotationTableNames
+                .compactMap { geodatabase.annotationTable(named: $0) }
+                .map(AnnotationLayer.init)
+            
+            // Adds the layers to the map.
+            map.addOperationalLayers(featureLayers)
+            map.addOperationalLayers(annotationLayers)
+        }
+        
+        /// Selects the first feature in a given list of identify layer results.
+        /// - Parameter results: The identify layer results.
+        func selectFirstFeature(from results: [IdentifyLayerResult]) {
+            clearSelectedFeature()
+            
+            // Gets the first feature from the first feature layer in the results.
+            let featureLayerResult = results.first { $0.layerContent is FeatureLayer }
+            guard let featureLayer = featureLayerResult?.layerContent as? FeatureLayer,
+                  let feature = featureLayerResult?.geoElements.first as? ArcGISFeature else { return }
```

Comment:
Not a big deal - when there are >= 2 optional chaining for a variable, it's better to `guard let` it first so there is less cost to unbox the optional.